src/utils/firebase/firebase.ts

```diff
@@ -0,0 +1,14 @@
+import { initializeApp } from "firebase/app";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyDyoMZH_jCmVOcmFG8vq9TLr2rO9sEsJHc",
+  authDomain: "nft-marketplace-7a526.firebaseapp.com",
+  projectId: "nft-marketplace-7a526",
+  storageBucket: "nft-marketplace-7a526.appspot.com",
+  messagingSenderId: "149855962248",
+  appId: "1:149855962248:web:712edf0fd07dae12fa0b3d",
+  measurementId: "G-1WBRRJLJKM",
+};
+
+const firebaseApp = initializeApp(firebaseConfig);
+export default firebaseApp;
```

Comment:
Prefer named exports

```suggestion
export const firebaseApp = initializeApp(firebaseConfig);
```